src/worker/main.ts

```diff
@@ -40,6 +40,10 @@ const getQueueOptions = (timeout: number): QueueOptions => {
 			removeOnComplete: true,
 			removeOnFail: true
 		},
+		settings: {
+			// lockDuration must be greater than the timeout, so that it doesn't get processed again prematurely
+			lockDuration: timeout + 500
```

Comment:
What's the default?

Reply:
The default is 30 seconds. And we set the timeout to all queues to 60 seconds or more. That means if a message took longer than 30 seconds, it wouldn't yet time out, but it would be freed up for the next worker to execute again.